### PR TITLE
ADEN-11341 Cleanup and upgrade Blockthrough integration

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts [hunk 1]
```diff
@@ -131,7 +131,10 @@ export class UcpDesktopDynamicSlotsSetup implements DiProcess {
 
 		container.id = slotName;
 		parentContainer.appendChild(container);
-		rotator.rotateSlot();
+
+		eventService.once(events.AD_STACK_START, () => {
```
Comment (preludek) [inline platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts:135]: `eventService` is deprecated, we should go with `communicationService`

File: platforms/ucp-desktop/templates/configs/outstream-config.ts [hunk 2]
```diff
@@ -1,11 +1,9 @@
 import { slotsContext } from '@platforms/shared';
-import { NavbarManager, PorvataTemplateConfig } from '@wikia/ad-engine';
+import { PorvataTemplateConfig } from '@wikia/ad-engine';
 
 export const getOutstreamConfig = (): PorvataTemplateConfig => {
-	const navbarManager = new NavbarManager(document.getElementById('globalNavigation'));
-
 	return {
-		inViewportOffsetTop: navbarManager.getHeight(),
+		inViewportOffsetTop: document.querySelector('.fandom-sticky-header')?.clientHeight || 0,
```
Comment (preludek) [inline platforms/ucp-desktop/templates/configs/outstream-config.ts:6]: Team nullish coalescing operator!

File: platforms/ucp-mobile/templates/configs/outstream-config.ts [hunk 3]
```diff
@@ -1,11 +1,9 @@
 import { slotsContext } from '@platforms/shared';
-import { NavbarManager, PorvataTemplateConfig } from '@wikia/ad-engine';
+import { PorvataTemplateConfig } from '@wikia/ad-engine';
 
 export const getOutstreamConfig = (): PorvataTemplateConfig => {
-	const navbarManager = new NavbarManager(document.querySelector('wds-global-navigation'));
-
 	return {
-		inViewportOffsetTop: navbarManager.getHeight(),
+		inViewportOffsetTop: document.querySelector('.wds-global-navigation')?.clientHeight || 0,
```
Comment (preludek) [inline platforms/ucp-mobile/templates/configs/outstream-config.ts:6]: ditto

**Review by preludek — CHANGES_REQUESTED** [review]

File: platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts (hunk 1 above)
Comment (dmnsobczak) [reply platforms/ucp-desktop/setup/dynamic-slots/ucp-desktop-dynamic-slots.setup.ts]: Replaced!

File: platforms/ucp-desktop/templates/configs/outstream-config.ts (hunk 2 above)
Comment (dmnsobczak) [reply platforms/ucp-desktop/templates/configs/outstream-config.ts]: Done!

File: platforms/ucp-mobile/templates/configs/outstream-config.ts (hunk 3 above)
Comment (dmnsobczak) [reply platforms/ucp-mobile/templates/configs/outstream-config.ts]: Done!

**Pull request merged**